src/enums/schemasMap.ts

```diff
@@ -23,9 +23,9 @@ export default {
   [services.authRoleAssignment]: 'azureAuthRoleAssignment',
   [services.authRoleDefinition]: 'azureAuthRoleDefinition',
   [services.autoProvisioningSettings]: 'azureAutoProvisioningSetting',
-  // [services.backupVault]: 'azureBackupVault',
-  [services.backupInstance]: 'azureBackupInstance',
-  [services.backupPolicy]: 'azureBackupPolicy',
+  [services.backupVaults]: 'azureBackupVaults',
```

Comment:
these values will break the gql api when you add in a rule set to evaluate. The schemas map needs to map the service name to the schema name but these have been pluralized. For example, the schema name for back up instance is `azureBackupInstance` not `azureBackupInstances`, so when it tries to extend that type to add in findings, it wont be able to find the type and throw an exception.

Reply:
oh yes, I've pluralized them by mistake, thanks for the catch!